### PR TITLE
feat: add new networks and clients

### PR DESCRIPTION
**Pull request by nick8319**
Pull request contains:

- Master Copies and Proxy Factories for 2 networks(https://github.com/safe-global/safe-deployments/pull/304):
  - ZETACHAIN_ATHENS_TESTNET
  - SCROLL

- addition of Blockscout clients for:
  - SHIDEN
  - SCROLL
  - SCROLL_SEPOLIA_TESTNET
  - ZETACHAIN_ATHENS_TESTNET
  - RSK_MAINNET
  - RSK_TESTNET
  - LINEA
  - LINEA_TESTNET
  - NEON_EVM_MAINNET
  - NEON_EVM_DEVNET
  - OASIS_SAPPHIRE
  - OASIS_SAPPHIRE_TESTNET
  - CASCADIA_TESTNET
  - TENET
  - TENET_TESTNET
  - VELAS_EVM_MAINNET
  - CRONOS_MAINNET_BETA
  - CRONOS_TESTNET
  - THUNDERCORE_MAINNET
  - THUNDERCORE_TESTNET

- addition of Etherscan clients for:
  - SCROLL
  - SCROLL_SEPOLIA_TESTNET

### CODE REVIEW

**Comment by ElvisKrop** [comment]
Hello guys! Please take a look at this PR whenever you can :pray: 
@Uxio0 @fmrsabino @hectorgomezv
We decided to open this single PR which includes tiny updates for quite many networks, in order not to spam you by opening separate PRs for each of them. Especially considering that merging each separate PR will lead to conflicts in all other similar PRs :upside_down_face: 
Thanks in advance!

**Comment by Uxio0** [comment]
Please let us know when safe-deployments related PRs are merged, but taking a look I see that for some networks you are not using the official expected addreses using the Safe singleton factory deployer

File: gnosis/safe/addresses.py [hunk 1]
```diff
@@ -595,6 +595,14 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2362236, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2362238, "1.3.0"),
     ],
+    EthereumNetwork.ZETACHAIN_ATHENS_TESTNET: [
+        ("0xfb1bffC9d739B8D520DaF37dF666da4C687191EA", 1962980, "1.3.0+L2"),
```
Comment (moisses89) [inline gnosis/safe/addresses.py:599]: @nick8319  I couldn't verify the deploy block using the blockexplorer https://zetachain-athens-3.blockscout.com/address/0xfb1bffC9d739B8D520DaF37dF666da4C687191EA 
How do you get this block?

**Review by moisses89 — COMMENTED** [review]
It is not a heavy request but if the block is wrongly configured you can lose Safes if there were any Safe created previously to that block.

File: gnosis/safe/addresses.py (hunk 1 above)
Comment (nick8319) [reply gnosis/safe/addresses.py:599]: Hey @moisses89, unfortunately Blockscout doesn't index internal transactions on Zetachain Testnet that's why it doesn't recognize `safe-contracts` as contracts. But I have transaction hashes saved during `safe-contracts` deployment. Here they are:
`GnosisSafe`: https://zetachain-athens-3.blockscout.com/tx/0x614a1428afd37470db6836abe53f191b3cc4470fd08212e0b3706166fcc24b44?tab=index
`GnosisSafeL2`: https://zetachain-athens-3.blockscout.com/tx/0x2934f55bae338f3af78fd2f86d98b42fd82f517feb9008b99837aaa4237b66e2?tab=index
`GnosisSafeProxyFactory`: https://zetachain-athens-3.blockscout.com/tx/0xfc3ba1aea3dd297d63172c4429bc59a47f6ddb0a1525ac156a6e83b8d6734cb9?tab=index

**Review by moisses89 — APPROVED** [review]

**Pull request merged**